apps/service-portal/src/hooks/useNavigation/useNavigation.ts

```diff
@@ -22,6 +23,12 @@ const filterNavigationTree = (
           route.path.includes(item.path)),
     ) !== undefined || item.systemRoute === true
 
+  // Only display the document-provider module for those who have access
+  if (item.path === '/skjalaveita') {
+    if (env.documentProviderAdmins?.indexOf(userInfo.profile.nationalId) === -1)
```

Comment:
Could also have used includes which I would prefer since its more readable and we dont really care about the index for this case..
```suggestion
    if (!env.documentProviderAdmins?.includes(userInfo.profile.nationalId))
```

Reply:
Good point... will fix